operations/support-money.rst

```diff
@@ -0,0 +1,125 @@
+.. _financni-podpora:
+
+Finanční podpora
+================
+
+.. contents:: Obsah
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+.. _financni-sluzby:
+
+Finanční služby
+---------------
+
+Přes Pyvec tečou peníze české Python komunity. Poskytujeme tyto hlavní služby:
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+
+.. _otoceni-penez:
+
+Otočení sponzorských peněz
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Jestliže na svou věc seženete sponzora, Pyvec mu pošle fakturu na potřebnou částku a peníze od sponzora uloží na svůj transparentní účet. Z těchto peněz vám věc oproti účetnímu dokladu :ref:`proplatí <jak-proplatit>`, a to bez nutnosti schvalování :term:`výborem <Výbor>`. Pokud z peněz něco zbude, zůstanou na účtu Pyvce pro :ref:`granty <granty>`. Příklad:
+
+   Jiřík každý měsíc :ref:`organizuje Python sraz <srazy>` v Zeleném Údolí. V Zeleném Údolí je bohužel jen jediný vhodný podnik, kde lze sraz konat, a to Hospoda na Mýtince. Ta si za akce účtuje pronájem 500 Kč za večer. Jiřík sežene sponzora srazu, místní firmu Červený Klobouk, a.s., která zaplatí pronájem na celý rok. Jiřík s touto dohodou seznámí Pyvec a poprosí jej, aby transakci vyřešil. Pyvec pošle Červenému Klobouku fakturu na 6000 Kč. Jiřík každý měsíc nájem zaplatí ze svého, doklad pošle Pyvci, a ten mu jej proplatí. Když to Jiříka přestane bavit, řekne Hospodě na Mýtince, aby rovnou poslala Pyvci fakturu za několik měsíců, a Pyvec ji proplatí.
+
+Na příkladu je vidět hned několik výhod tohoto modelu:
+
+- Sponzor jen pošle peníze a nemusí se dál o nic starat.
+- Peníze lze přijmout najednou i v případě, že se potom vyplácí postupně.
+- Jiřík není na sponzorovi závislý. Hospoda na Mýtince jedná s Pyvcem, sponzoři se mohou měnit.
+- Sponzorů může být i více než jeden a Hospoda na Mýtince stále jedná pouze s Pyvcem.
+- Kdyby byla ve hře např. nájemní smlouva, podepisuje ji Pyvec, ne sponzor.
+- Jiřík má za sebou právní entitu, která je schopna přijímat nebo vydávat faktury a má transparentní účet.
+
+.. note::
+   Pyvec může věci :ref:`proplácet <jak-proplatit>` jen oproti účetnímu dokladu, tzn. účtence nebo faktuře. Pokud vám sponzor pošle více peněz než nakonec bylo potřeba, nelze vám je jen tak vyplatit. Můžete je ale v budoucnu čerpat na další věci pomocí :ref:`grantů <granty>`.
+
+
+.. _granty:
+
+Granty
+^^^^^^
+
+Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů od náhodných lidí nebo firem, a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__.
```

Comment:
```suggestion
Pokud na svou věc nemáte sponzora, můžete požádat Pyvec o grant. Pyvec disponuje penězi, které na jeho účtě zbyly z nevyčerpaných sponzorských darů, z neurčených darů od náhodných lidí nebo firem a z přebytků hospodaření konference `PyCon CZ <https://cz.pycon.org/>`__.
```

Viz [Psaní čárky před spojkami a, i, ani](https://prirucka.ujc.cas.cz/?id=153) (1.1. Poměr slučovací).
Tohle mi tak dlouho opravoval v materiálech Miro, až jsem si navykl na to dávat pozor :)